src/frontend/Komponenter/Behandling/Inngangsvilkår/NyttBarnSammePartner/RegistergrunnlagNyttBarn.tsx

```diff
@@ -20,50 +16,41 @@ const RegistergrunnlagNyttBarnInnhold: FC<Props> = ({ barn }) => {
 
     const ikkeOppgittAnnenForelderBegrunnelse = barn.ikkeOppgittAnnenForelderBegrunnelse;
     return (
-        <GridTabell>
-            <>
-                <LiteBarn />
-                <Label size={'small'} as={'div'}>
-                    {utledNavnOgAlder(navn, fødselsdato, dødsdato)}
-                    {barn.dødsdato && <EtikettDød dødsdato={barn.dødsdato} />}
-                </Label>
-            </>
-            <>
-                <Registergrunnlag />
-                <BodyShortSmall>Fødsels eller D-nummer</BodyShortSmall>
-                <BodyShortSmall>{barn.fødselsnummer}</BodyShortSmall>
-            </>
-
+        <BarneInfoWrapper
+            navnOgAlderPåBarn={utledNavnOgAlder(navn, fødselsdato, dødsdato)}
+            dødsdato={barn.dødsdato}
+        >
+            <Informasjonsrad
+                ikon={VilkårInfoIkon.REGISTER}
+                label="Fødsels eller D-nummer"
+                verdiSomString={false}
+                verdi={
+                    barn.fødselsnummer && (
+                        <KopierbartNullableFødselsnummer fødselsnummer={barn.fødselsnummer} />
+                    )
+                }
+            />
             {annenForelderRegister && (
-                <>
-                    <Registergrunnlag />
-                    <BodyShortSmall>Annen forelder fra folkeregister</BodyShortSmall>
-                    <AnnenForelderNavnOgFnr forelder={annenForelderRegister} />
-                </>
+                <Informasjonsrad
+                    ikon={VilkårInfoIkon.REGISTER}
+                    label="Annen forelder fra folkeregister"
+                    verdiSomString={false}
+                    verdi={<AnnenForelderNavnOgFnr forelder={annenForelderRegister} />}
+                />
             )}
 
             {harVerdi(ikkeOppgittAnnenForelderBegrunnelse) && (
-                <>
-                    <Søknadsgrunnlag />
-                    <BodyShortSmall>Annen forelder</BodyShortSmall>
-                    <BodyShortSmall>
-                        {ikkeOppgittAnnenForelderBegrunnelse === 'donorbarn'
+                <Informasjonsrad
+                    ikon={VilkårInfoIkon.SØKNAD}
+                    label="Annen forelder"
+                    verdi={
+                        ikkeOppgittAnnenForelderBegrunnelse === 'donorbarn'
                             ? ikkeOppgittAnnenForelderBegrunnelse
-                            : `Ikke oppgitt: ${ikkeOppgittAnnenForelderBegrunnelse}`}
-                    </BodyShortSmall>
-                </>
-            )}
-
-            {barn.annenForelderRegister?.dødsfall && (
-                <>
-                    <Registergrunnlag />
-                    <BodyShortSmall>Annen forelder dødsdato</BodyShortSmall>
-                    <BodyShortSmall>
-                        {formaterNullableIsoDato(barn.annenForelderRegister.dødsfall)}
-                    </BodyShortSmall>
-                </>
```

Comment:
Her ble annen forelder sitt dødsdato fjernet?

Reply:
`AnnenForelderNavnOgFnr` legger på en etikett om personen er død med dødsdato. Denne ble da fjernet for å slippe dobbel informasjon